public/test/setupTests.ts

```diff
@@ -1,14 +1,9 @@
 import '@testing-library/jest-dom';
 import i18next from 'i18next';
-import failOnConsole from 'jest-fail-on-console';
 import { initReactI18next } from 'react-i18next';
 
 import { matchers } from './matchers';
 
-failOnConsole({
-  shouldFailOnLog: true,
-});
-
```

Comment:
think this removal was left in by accident 😬

Reply:
😬 

tbh, i now think we should remove this from local dev, and have it only in CI

Reply:
we did - it must have got reverted at some point. if you look in the file history it definitely only used to do this in CI

Reply:
lol